lib/booktype/convert/epub/writer.py

```diff
@@ -51,6 +51,188 @@ def _write_container(self):
             display_options_xml
         )
 
+    def _write_opf_file(self):
```

Comment:
We should avoid doing this. I think it would be better to improve/refactor this method on ebooklib in order to make it easier to extend instead of repeating almost the whole code of the base method. Let's send PR to ebooklib and then improve this one. Let's keep the PR open though.

Reply:
@eos87 , the thing is that changes I've done are only booktype related, so providing fix directly to the library is not good option. What I can do, I can try to create some kind of hook in the library which will not brake/change all opf writing process and implement it inside booktype.